src/api/wix/WixToolset.Data/Symbols/WixPackageSymbol.cs

```diff
@@ -44,12 +44,9 @@ public enum WixPackageAttributes
     {
         None = 0x0,
         PerMachine = 0x1,
-    }
-
-    public enum WixPackageUpgradeStrategy
-    {
-        None = 0x0,
-        MajorUpgrade = 0x1,
+        PerUser = 0x2,
```

Comment:
Is this a general approach, to store bit flags in an `Attributes`?

Reply:
Yeah, we do that in several symbols. The alternative is a lot of bools. The important part is that the attributes are combined correctly. Previously, the UpgradeStrategy was stomping over the other attributes. We just didn't notice because `per-machine` and `major ugprade` were both `0x1`. 🤬

Reply:
I've thought about this more and decided that using `Attributes` for these two fields is too error-prone because two enum concepts are being squished into a single-bit field. So, I'll introduce dedicated fields for scope and upgrade strategy.

Reply:
Hey, we're not MSI, needing to bit-twiddle to fit into the registry... :)